Terraform/grafana-log.tf

```diff
@@ -0,0 +1,22 @@
+resource "azurerm_monitor_diagnostic_setting" "grafana-diagnostics" {
+  name                       = var.gf_diag_name[terraform.workspace]
+  target_resource_id         = azurerm_linux_web_app.grafana-web-app.id
+  log_analytics_workspace_id = data.azurerm_log_analytics_workspace.log-analytics-ws.id
+
+  metric {
```

Comment:
Do we need this enabled? Does this mean ALL metrics sent (which is not what we want)

Reply:
It's metrics though not logs.  Do you not want any metrics at all?

Reply:
ok let deploy and when done I can review dev. thanks